jython/loconet/PGM_DS74_GEN.py

```diff
@@ -0,0 +1,469 @@
+# SCRIPT TO PROGRAM DS76 # GEN
```

Comment:
No need to change, but it would be easier to find this script if the file name was changed to something like `Program_DS74_ByTextFile.py`.

Reply:
Sorry, this is my first post.  I just used my developed name.  Will do better next time.

Reply:
No problem at all! Thanks for providing this script!